src/content/docs/infrastructure/host-integrations/host-integrations-list/stripe-integration.mdx

```diff
@@ -0,0 +1,140 @@
+---
+title: Stripe integration
+tags:
+  - Stripe integration
+  - New Relic integration
+metaDescription: Install our Stripe dashboards and see your Stripe data in New Relic.  
+---
+
+import infrastructureStripeDashboard from 'images/infrastructure_screenshot-stripe-dashboard.webp'
+
+Monitor your Stripe application in a single, pre-built Stripe dashboard that tracks metrics like transaction overview, transaction volume, revenue, and transaction errors. Simply install our agents to collect your app data, and you'll be able to see all of your Stripe app data in one place.
+
+<img
+    title="Stripe"
+    alt="Stripe dashboard"
+    src={infrastructureStripeDashboard}
+/>
+
+<figcaption>
+View all your Stripe app's metrics in one dashboard.
+</figcaption>
+
+## Set up Stripe monitoring
+
+<Steps>
+  <Step>
+        ### Install the infrastructure agent [#infra]
+
+Install our infrastructure agent to get details about your host, such as CPU usage, memory usage, storage usage, network traffic, and more.
+
+To install the agent, follow the [guided install](https://one.newrelic.com/nr1-core?state=999174fa-ed8b-d707-af7b-8e4f8fd03cc1) or see the [Infrastructure agent installation docs](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent/) to install manually.  
+
+    </Step>
+  <Step>
+    ### Install an APM agent [#apm-setup]
+
+Install one of our application performance monitoring (APM) agents in the [guided install](https://one.newrelic.com/nr1-core?state=999174fa-ed8b-d707-af7b-8e4f8fd03cc1).
+    </Step>
+
+  <Step>
+    ### Install our browser monitoring agent [#browser-setup]
+
+Our browser monitoring agent reports application and browser-related metrics like AJAX performance, error rate, duration, core web vitals, and throughput.
+* See [Install the browser monitoring agent](/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent/) to get started.
+    </Step>
+
+    <Step>
+      ### Set up synthetic monitoring [#synthetic-setup]
+
+Our synthetic monitors test your app in different locations and capture server downtime. You can view this data in the Stripe dashboard.
+
+To set up synthetic monitors, follow the steps in our [guided install](https://onenr.io/0EjOXrnxgQ6)
+  </Step>
+
+    <Step>
+### Send metrics to New Relic [#send-metrics]
+
+Check the [Metric API documentation](/docs/data-apis/ingest-apis/metric-api/report-metrics-metric-api/) for further details.
+
+To proceed further, you will need a New Relic license key. See [New Relic API Keys](/docs/apis/intro-apis/new-relic-api-keys/) to get your license key.
+On a successful stripe payment, we receive a JSON from the stripe API service. We need to send these values to New Relic by formatting the JSON as per New Relic Metric API requirements.
```

Comment:
@sjyothi54 Should this be renamed as an optional step to set up custom metrics? 
```suggestion
### (Optional) Send custom metrics to New Relic [#send-metrics]

If you need to customize the app metrics sent to New Relic, see [Metric API documentation](/docs/data-apis/ingest-apis/metric-api/report-metrics-metric-api/) to learn how. Once you've set up custom metrics, with a successful stripe payment, we receive a JSON from the stripe API service. We need to send these values to New Relic by formatting the JSON as per New Relic Metric API requirements.

If you're sending custom metrics, you'll need a New Relic license key. See [New Relic API Keys](/docs/apis/intro-apis/new-relic-api-keys/) to get your license key.
```

Reply:
@ally-sassman This step is essential, not optional.